authors/apps/articles/serializers.py

```diff
@@ -32,6 +34,28 @@ def to_representation(self, instance):
         # return the article's details along with it's read time
         return article_rep
 
+    def get_share_links(self, obj):
+        links = {}
+
+        title = obj.title
+        description = obj.description
+
+        article_link = reverse('articles:article', kwargs={'slug': obj.slug})
+
+        request = self.context.get('request')
+
+        if request:
+            article_link = parse.quote(request.build_absolute_uri(article_link))
+            title = parse.quote(title)
+            description = parse.quote(description)
+
+            links['facebook'] = "https://www.facebook.com/sharer/sharer.php?u={}".format(article_link)
+            links['google_plus'] = "https://plus.google.com/share?url={}".format(article_link)
```

Comment:
Remove the google plus link as google plus is officially shutting down personal and brand accounts.